modules/ROOT/pages/workflow-sharing-assets.adoc

```diff
@@ -1,16 +1,15 @@
-= Workflow: Share Assets
+= Workflow: Publish Assets
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Anypoint Exchange provides the ability to share assets in Exchange private and Exchange portal.
+Anypoint Exchange provides the ability to publish assets in Exchange private and Exchange portal.
```

Comment:
"public and private Exchange portals"? 
Standardize GLOBALLY.

Reply:
Fixed.